docs/REST-API/fliplet-datasources.md

```diff
@@ -390,8 +392,8 @@ The following sample request applies the following changes to the data source:
     { "data": { "email": "alice@example.org" } }
   ],
   "delete": [456],
-  "append": true,
-  "extend": true,
+  "append": "true",
```

Comment:
@debankurghosh2061 Is it really the case that these have to be strings? They look like they should work as boolean values rather than strings (which are always truthy).

Reply:
@tonytlwu You are correct, I will remove this change. It shouldn't be a string. It seems that passing it as a string causes them to be false and hence any existing entries are removed.